README.markdown

```diff
@@ -58,12 +58,7 @@ To build xen-api from source, we recommend using [opam](https://opam.ocaml.org/d
 6) Install all the Packages.
 
     ```bash
-    PACKAGES="xapi-cli-protocol xapi-client xapi-consts xapi-datamodel xapi-types xapi xe xen-api-sdk xen-api-client xen-api-client-lwt xen-api-client-async xapi-rrdd xapi-rrdd-plugin xapi-rrd-transport xapi-rrd-transport-utils rrd-transport rrdd-plugin rrdd-plugins rrddump gzip http-lib pciutil safe-resources sexpr stunnel uuid xapi-compression xml-light2 zstd vhd-tool xs-toolstack"
-
-    # NOT needed with opam>=2.1.0) Install all the dependencies (Including OS):
-    opam --yes depext --yes -u $PACKAGES # The first '--yes' is to install depext itself
-    # Install the Packages finally:
-    opam install $PACKAGES --yes --deps-only --with-test -v
+    opam install xs-toolstack
```

Comment:
do we want `--yes --deps-only --with-test -v` at all?

Reply:
If you look at the dependencies of `xs-toolstack`, it is the list of all components in the toolstack but itself has no code. The result is that all toolstack components are installed. This might not be desired for development because we only would need dependencies but it would problems and is simple - hence why I like it.

I don't know what effect `--with-test` has in this context because it usually refers to dependencies of the immediate code (which we don't have here). Previously we denoted packages with code, where the flags made sense. 

Using `--yes` would skip over some confirmations - but there are not so many that hey are annoying.

Reply:
The change here is not equivalent, now the dependencies to run the unit-tests are not installed.

One that would work is `opam list -s --required-by xs-toolstack | xargs opam install -t`